### ISSUE TITLE
how to train hourglass model??

### ISSUE THREAD

**Issue by lvchigo** [edvardHua/PoseEstimationForMobile#16]
From this(PoseEstimationForMobile/training/src/networks.py) file ,we just can see mv2_cpm config. So, how to train hourglass  @model??
`
import network_mv2_cpm

def get_network(type, input, trainable=True):
    if type == 'mv2_cpm':
        net, loss = network_mv2_cpm.build_network(input, trainable)
    return net, loss
`

**Comment by edvardHua** [comment]
Oh, it similar to mv2_cpm. Pls check the [networks.py](https://github.com/edvardHua/PoseEstimationForMobile/blob/master/training/src/networks.py) again.
